plugin.video.alfa/channels/videolibrary.py

```diff
@@ -10,10 +11,43 @@
 from platformcode import config, logger
 from platformcode import platformtools
 
+def videolibrary_check_format():
+    logger.info()
+
+    if config.get_setting("videolibrary_check_flag") == None:
+
+        p_dialog = platformtools.dialog_progress_bg("Alpha", "Converting the library to the new format...")
+        p_dialog.update(0)
+
+        import glob
+        movies_nfos = glob.glob(filetools.join(videolibrarytools.MOVIES_PATH, u'/*/*.nfo'))
+        tvshows_nfos = glob.glob(filetools.join(videolibrarytools.TVSHOWS_PATH, u'/*/*.nfo'))	
+        num_nfos = len(movies_nfos) + len(tvshows_nfos)
+        counter =  0
+
+        for filename in movies_nfos:
+            counter += 1
+            p_dialog.update((counter*100)/num_nfos)
+            os.rename(filename, filename.replace('.nfo', '.info'))
+
+        for filename in tvshows_nfos:
+            counter += 1
+            p_dialog.update((counter*100)/num_nfos)
+            os.rename(filename, filename.replace('.nfo', '.info'))
+
+        xbmc.sleep(2000)

```

Comment:
Si peta la mediaserver tal vez lo podrías poner como platformtools y en xbmc se pone está línea y en mediaserver pones un sleep

Reply:
See my previous comment. Btw, that sleep is used just to keep the notification on the screen for two seconds after the library conversion is finished. It is not strictly necessary then.